### ISSUE TITLE
error in loss function

### ISSUE THREAD

**Issue by WynMew** [znxlwm/pytorch-CartoonGAN#10]
in pre-training, your recon loss:
~~~
Recon_loss = 10 * L1_loss(G_feature, x_feature.detach()) 
~~~
the '10' here is useless.

**Comment by xudongwang0828** [comment]
10 is a parameter which proposed by authors.It's used to balance generate pic's style and content.You can read the paper.
